<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="users-MacBook-Pro.local" failures="0" tests="12" name="Page_Test.Refer_a_friend_and_Contact_us_test" time="216.769" errors="1" timestamp="2024-08-12T10:20:06 IST" skipped="0">
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC054_Refer_a_friend" time="20.571"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC055_Verify_Select_State" time="16.064"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC056_Verify_Select_City" time="5.713"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC057_Verify_Select_Model" time="25.424"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC058_Verify_Refer_yourself" time="53.171"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC059_Verify_Facebook_in_Contact_us_page" time="27.479"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC060_Verify_instagram_in_Contact_us_page" time="22.135"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC061_Verify_Whatsapp_in_Contact_us_page" time="1.406"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC062_Verify_youTube_in_Contact_us_page" time="8.336"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC063_Verify_Linkedin_in_Contact_us_page" time="23.603"/>
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC064_Verify_twitter_in_Contact_us_page" time="0.941">
    <error message="Cannot invoke &quot;com.utility.listner.getcapcture(String)&quot; because &quot;Page_Test.Refer_a_friend_and_Contact_us_test.lis&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.utility.listner.getcapcture(String)" because "Page_Test.Refer_a_friend_and_Contact_us_test.lis" is null
at Page_Test.Refer_a_friend_and_Contact_us_test.TC064_Verify_twitter_in_Contact_us_page(Refer_a_friend_and_Contact_us_test.java:167)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- TC064_Verify_twitter_in_Contact_us_page -->
  <system-out/>
  <testcase classname="Page_Test.Refer_a_friend_and_Contact_us_test" name="TC065_Verify_Contact_via_Email" time="11.926"/>
  <system-out/>
</testsuite> <!-- Page_Test.Refer_a_friend_and_Contact_us_test -->
